### PR TITLE
test: fix merge skew on cloudtest

### PR DESCRIPTION
**Pull request by andrioni**
#15002 was merged using syntax that was changed by #15019, and a breaking change in `CREATE SOURCE` introduced by #14597.

### Motivation

  * This PR fixes a previously unreported bug: merge skew breaking `main` after #15002 got merged.

### CODE REVIEW

**Review by jkosh44 — APPROVED** [review]
LGTM

**Comment by philip-stoev** [comment]
I am sorry for the conflict -- I had to make this change in a hurry today to unblock Antithesis.

**Comment by andrioni** [comment]
No worries, it's my fault that I forgot to merge with main before merging my PR!

**Pull request merged**
